raiden/exceptions.py

```diff
@@ -163,8 +163,5 @@ class DuplicatedChannelError(RaidenError):
 
 class TransactionThrew(RaidenError):
     """Raised when, after waiting for a transaction to be mined,
-    the gasUsed in receipt is the same as the provided transaction gas limit"""
-    def __init__(self, txname, receipt):
-        super(TransactionThrew, self).__init__(
-            '{} transaction threw. Receipt={}'.format(receipt)
-        )
+    the gasUsed in receipt is the same as the provided transaction gas limit.
```

Comment:
Can you instead fix the original problem and not remove the arguments from here as we discussed on rocketchat?
Just add the other missing `txname` argument to the format string.

Reply:
done